mathics/format/tex.py

```diff
@@ -0,0 +1,559 @@
+# -*- coding: utf-8 -*-
+"""
+Format a Mathics object as mathml code
```

Comment:
I suspect this comment a cut and paste mistake. It should be "tex", but better would be LaTeX. 

I know we get to this in Mathics via TeXForm of something like this. But TeX is not LaTeX.  And similarly this module should be renamed latex.py not tex.py.

I don't get why in this project there is a tendency to use at best vague terms, or at works misleading, confusing or wrong terms.

Reply:
Thanks for noticing this. Regarding TeX/LaTex, (as in many other places) the name was chosen trying to follow the WL convention:   WL has `TeXForm` no `LaTeX` form. Maybe because of historical reasons (the first versions of TeXForm in WMA comes from 1988). Also, most of the commands produced in `TeXForm`  are almost pure *TeX* code (*LaTeX* is a superset of macros defined over *TeX* ).

Reply:
The typo is fixed in #526. Maybe we could also add this comment.